polymer.externs.js

```diff
@@ -443,6 +443,12 @@ PolymerDomApi.prototype.removeAttribute = function(attribute) {};
 PolymerDomApi.prototype.classList;
 
 /**
+ * @param {string} selector
+ * @return {!Array<?HTMLElement>}
```

Comment:
I can't think of any reasonable scenario. However, this is also the annotation used for querySelectorAll (above), which leads me to think it might be possible. From my reading of https://github.com/Polymer/polymer/blob/ec4f4976ea7180c97b76fd9b0986ba886e12f077/src/lib/dom-api.html#L398, a null entry *could* make it into childNodes, but this is unlikely.